include/server/api/route_parameters_grammar.hpp

```diff
@@ -42,6 +42,13 @@ struct RouteParametersGrammar : public BaseParametersGrammar<Iterator, Signature
 
     RouteParametersGrammar(qi::rule<Iterator, Signature> &root_rule_) : BaseGrammar(root_rule_)
     {
+        const auto add_annotation = [](engine::api::RouteParameters &route_parameters,
```

Comment:
this lambda is not needed with `operator |=` and in-place phoenix bind (i added related comments)